avocado/plugins/run.py

```diff
@@ -152,8 +152,11 @@ def configure(self, parser):
                              help="Inject [path:]key:node values into the "
                              "final multiplex tree.")
 
-    def _activate(self, args):
-        # Extend default multiplex tree of --mux_inject values
+    @staticmethod
+    def _process_mux_inject(args):
```

Comment:
Actually, @ldoktor is there any reason for those two (from `plugins/run.py` and from `plugins/multiplex.py`) to differ?